Appveyor tests are stuck on current master

Appveyor builds are stuck on when running on current master 06677a9e10b3bb6c2dc2e54bc137e72b6f39640d. The build is stuck on the python 3.5 tests using pyqt5.

locally on a windows 10 machine the python 3.5 pyqt5 tests run and fail with the following traceback

```python
======================================================================
FAIL: traitsui.tests.test_splitter_prefs_restored.test_splitter_prefs_are_restored
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\...\.edm\envs\traitsui-test-3.5-pyqt5\lib\site-packages\nose\case.py", line 198, in runTest
    self.test(*self.arg)
  File "C:\Users\...\.edm\envs\traitsui-test-3.5-pyqt5\lib\site-packages\traitsui-5.2.0.dev356-py3.5.egg\traitsui\tests\_tools.py", line 88, in wrapper
    return item(*args, **kwargs)
  File "C:\Users\...\.edm\envs\traitsui-test-3.5-pyqt5\lib\site-packages\traitsui-5.2.0.dev356-py3.5.egg\traitsui\tests\test_splitter_prefs_restored.py", line 136, in test_splitter_prefs_are_restored
    _get_nattr(ui2.get_prefs()))
AssertionError: b'\x0[54 chars]x00\x00\x00\x00\x02M\x01\xff\xff\xff\xff\x00\x[14 chars]\x01' != b'\x0[54 chars]x00\x89\x00\x00\x02"\x01\xff\xff\xff\xff\x00\x[14 chars]\x01'
```

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,5 +1,5 @@
 build: false
-shallow_clone: true
+shallow_clone: false
 skip_branch_with_pr: true
 environment:
 

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -5,7 +5,7 @@ environment:
 
   global:
     PYTHONUNBUFFERED: "1"
-    INSTALL_EDM_VERSION: "1.7.1"
+    INSTALL_EDM_VERSION: "1.9.1"
 
   matrix:
 

--- a/etstool.py
+++ b/etstool.py
@@ -143,7 +143,7 @@ def install(runtime, toolkit, environment):
         "edm run -e {environment} -- python setup.py install"]
     # pip install pyqt5, because we don't have it in EDM yet
     if toolkit == 'pyqt5':
-        commands.append("edm run -e {environment} -- pip install pyqt5")
+        commands.append("edm run -e {environment} -- pip install pyqt5<=5.9.2")
 
     click.echo("Creating environment '{environment}'".format(**parameters))
     execute(commands, parameters)

--- a/etstool.py
+++ b/etstool.py
@@ -143,7 +143,7 @@ def install(runtime, toolkit, environment):
         "edm run -e {environment} -- python setup.py install"]
     # pip install pyqt5, because we don't have it in EDM yet
     if toolkit == 'pyqt5':
-        commands.append("edm run -e {environment} -- pip install pyqt5<=5.9.2")
+        commands.append("edm run -e {environment} -- pip install pyqt5==5.9.2")
 
     click.echo("Creating environment '{environment}'".format(**parameters))
     execute(commands, parameters)

